README.md

```diff
@@ -447,8 +447,10 @@ Provides volume levels and state of requested mixers.
 Supported platforms: Linux (required tool: amixer), FreeBSD.
 
 - Arguments (per platform):
-  * Linux: takes the ALSA mixer control as an argument, i.e. `"Master"`,
-    optionally append the card ID or other options, i.e. `"PCM -c 0"`
+  * Linux: takes either a single argument containing the ALSA mixer control as
+    an argument, i.e. `"Master"`, or a table, with the first argument being the
+    ALSA mixer control and the second being the ALSA device ID, i.e.
+    `{"Master", "pulse"}`.
```

Comment:
Btw. we have also a dedicated pulseaudio widget:
https://github.com/Mic92/vicious/blob/master/contrib/pulse_all.lua